### PR TITLE
Front refactoring

### PR DESCRIPTION
**Pull request by GuskoMaksim**
Changing the code to the general code style

### CODE REVIEW

File: Client/Client.Qt/Widgets/MessageWidget.hpp [hunk 1]
```diff
@@ -57,9 +60,9 @@ class MessageWidget : public QWidget
     void setStdTime_tDateTime(std::time_t newDataTime);
     /**
      * @brief Method for changing reaction map.
-     * @param new reactions of map with kay std::string and value int.
+     * @param new reactions of map with kay std::int and value int.
```
Comment (HRenata) [inline Client/Client.Qt/Widgets/MessageWidget.hpp:63]: int without std::

File: Client/Client.Qt/Widgets/MessageWidget.hpp [hunk 2]
```diff
@@ -102,6 +105,14 @@ private slots:
     void reactionChange(int index);
 
 private:
+    enum reactions
```
Comment (HRenata) [inline Client/Client.Qt/Widgets/MessageWidget.hpp:108]: CamelCase

File: Client/Client.Qt/Widgets/MessageWidget.hpp [hunk 3]
```diff
@@ -102,6 +105,14 @@ private slots:
...
+    {
+        like,
+        dislike,
+        fire,
+        cat,
+        Non
```
Comment (HRenata) [inline Client/Client.Qt/Widgets/MessageWidget.hpp:114]: CamelCase/ALL_CAPS

File: Client/Client.Qt/Widgets/MessageWidget.cpp [hunk 4]
```diff
@@ -77,10 +77,10 @@ void MessageWidget::initializationUiNotDelete()
         reactionMapLabel->insert(i, new Label);
         reactionMapLabelIcon->insert(i, new Label);
     }
-    pixmapIcon->insert(reactions::like, new QPixmap(":/reactions/like.png"));
-    pixmapIcon->insert(reactions::dislike, new QPixmap(":/reactions/dislike.png"));
-    pixmapIcon->insert(reactions::fire, new QPixmap(":/reactions/fire.png"));
-    pixmapIcon->insert(reactions::cat, new QPixmap(":/reactions/cat.png"));
+    pixmapIcon->insert(reactions::LIKE, new QPixmap(":/reactions/LIKE.png"));
```
Comment (HRenata) [inline Client/Client.Qt/Widgets/MessageWidget.cpp:80]: name of file not ALL_CAPS)

**Review by HRenata — APPROVED** [review]

**Review by Svidunovich — APPROVED** [review]

**Pull request merged**